torch/_dynamo/variables/torch.py

```diff
@@ -446,6 +448,29 @@ def handle_numel(self, tx: "InstructionTranslator", input):
                 # Workaround dynamic shapes issue
                 return input.call_method(tx, "numel", [], {})
 
+        @register(torch.compile)
+        def handle_torch_compile(self, tx: "InstructionTranslator", *args, **kwargs):
+            # torch.compile is a no-op in dynamo
+            user_stack = torch._guards.TracingContext.extract_stack()
+            user_stack_formatted = "".join(traceback.format_list(user_stack))
+            log.warning(
```

Comment:
I don't know if we should throw this warning - seems excessive. What is the user gonna do about it? In many cases the double wrapped compile seems like very reasonable behavior.

Reply:
Yes, warning is not actionable here. I think what I want is then for TorchDynamo to check if the inner compile `backend` is same as outer compile `backend`. If not, then graph break. This might be tricky to implement, but from user perspective that seems like the right thing to do.